Task1(Express)/app.js

```diff
@@ -0,0 +1,192 @@
+const express = require("express");
+const app = express();
+// var bodyParser = require('body-parser');
+app.use(express.json());
+const mongoose = require("mongoose");
+const port = 5000;
+
+mongoose
+  .connect("mongodb://localhost:27017/demodb")
+  .then(() => {
+    console.log("Connection Successful...");
+  })
+  .catch(() => {
+    console.log("Connection is not stablished , TRY again..");
+  });
+
+// Creating mongoose schema
+
+const demodataschema = new mongoose.Schema({
+  _score: Number,
+  category: String,
+  id: Number,
+  title: String,
+  timestamp: String,
+});
+
+// A mongoose schema defines the structure of the document, default values
+// , validators , etc..whereas a Mongoose model provides an interface to the
+// database for creating ,querying ,updating and deleting records
+
+// or collection creation
+const Data = new mongoose.model("Data", demodataschema);
+
+// //create document or insert a document
+
+const createDocument = async () => {
+  try {
+    const second_data = new Data({
+      _score: 3,
+      category: "publications",
+      id: 7,
+      title:
+        "Pissarro Paintings and Works on Paper at the Art Institute of Chicago",
+      timestamp: "2022-10-03T16:40:03-05:00",
+    });
+    const third_data = new Data({
+      _score: 10,
+      category: "publications",
+      id: 12,
+      title: "The Modern Series at the Art Institute of Chicago",
+      timestamp: "2022-10-03T16:40:03-05:00",
+    });
+    const fourth_data = new Data({
+      _score: 2,
+      category: "publications",
+      id: 406,
+      title: "Whistler and Roussel: Linked Visions",
+      timestamp: "2022-10-03T16:40:03-05:00",
+    });
+    const fifth_data = new Data({
+      _score: 5,
+      category: "publications",
+      id: 226,
+      title: "James Ensor: The Temptation of Saint Anthony",
+      timestamp: "2022-10-03T16:40:03-05:00",
+    });
+
+    const result = await Data.insertMany([third_data, fourth_data, fifth_data]);
```

Comment:
This is not the apt way to insert multiple documents into a mongoose. Check the correct usage here: https://mongoosejs.com/docs/api/model.html#model_Model-insertMany